.gaplint.yml

```diff
@@ -1,5 +1,4 @@
 disable:
-- W000
```

Comment:
I think these `.gaplint.yml` are not possible while there are still files in `hpcgap/lib/` which are not de-tabbed

Reply:
I've restarted the gaplint job. If it finishes, please squash as described.

The `testexpect` failure is indeed separate, see issue #5309

Reply:
Will squash now thanks!

Reply:
Squashed, hopefully correctly.